includes/class-sensei-course-participants-widget.php

```diff
@@ -168,7 +168,7 @@ public function update ( $new_instance, $old_instance ) {
 		$instance = $old_instance;
 
 		/* Strip tags for title and limit to remove HTML (important for text inputs). */
-		$instance['title'] = strip_tags( $new_instance['title'] );
+		$instance['title'] = esc_html( $new_instance['title'] );
```

Comment:
This changes functionality rather than merely adding escaping, I think. Is it a desired change?